.gitignore

```diff
@@ -39,3 +39,7 @@ yarn-error.log*
 junit.xml
 
 *.tsbuildinfo
+
+/.pnp.js
+/.yarn/
```

Comment:
we might wanna check this stuff in, I dunno... there were _so_ many files

Reply:
We need at least `.pnp.js` because that's where file sources are. The idea of "zero install" is for the deps to be checked in in a js file, so we actually may need to check in more, but not sure how much, didn't play with it yet

Reply:
If you want to check in `.pnp.js` you'd need `.yarn/cache` as well.
`.pnp.js` is the map to the cache, without the cache there is no point to check it in

Reply:
I think that you shouldn't ignore `.yarn/releases/yarn-berry.js`, because:
1. It ensures that everyone is using the same yarn version
2. If you make a fresh clone of the repository without that file, yarn won't work because `.yarnrc` is pointing to a non-existent file.

Also, `.pnp.js` should only be committed if you want "zero installs", but then you should commit the whole `.yarn` folder.

Reply:
should be helpful https://stackoverflow.com/a/58550620/11895808

Reply:
Or the documentation https://yarnpkg.com/advanced/qa/#which-files-should-be-gitignored

Reply:
Thanks! ca8bf7c8b7a6f295712250c0867e813a8ef8cc85